### PR TITLE
add phenograph 1.5.7

### PR DESCRIPTION
**Pull request by leej3**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

There is some binary repackaging going on here (with binaries built 7 years ago). I wish to reproduce the pypi released package which also distributes the binaries from the release tar.

### CODE REVIEW

**Comment by leej3** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c

File: recipes/phenograph/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PhenoGraph-{{ version }}.tar.gz
+  sha256: 46b28f9e043a00deba53bb5f35dd84793669ab2bd4ce78900bf7f15f1321515a
+
+build:
+  skip: True  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  missing_dso_whitelist:
+    - /lib64/*
```
Comment (xhochy) [inline recipes/phenograph/meta.yaml:17]: Please remove this.

File: recipes/phenograph/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - leidenalg
+    - igraph
+    - libgcc-ng
```
Comment (xhochy) [inline recipes/phenograph/meta.yaml:26]: This should not be needed. Is this binary repackaging?

**Review by xhochy — CHANGES_REQUESTED** [review]
It seems like this is repackaging binaries. These should be built from source instead.

**Comment by leej3** [comment]
Thanks for the review. I will close this PR. I have since learned this is likely not a good idea to introduce to the conda ecosystem. The c++ functionality will not evolve without the source code and there may be vulnerabilities introduced into the ecosystem from the compiler flags/optimizations that were used at build time that we can't control. 

I contacted one of the authors of the C++ code (resulting in the binaries being repackaged here). They suggested there are better implementations of this functionality:
+ More recent, faster, simpler interface (but no weighted graphs): https://github.com/jlguillaume/louvain. 
+ Older code that maintains a similar interface (implements weighted graphs) to the Phenograph binaries: https://sourceforge.net/projects/louvain
 
For language bindings in higher level languages one could consider:
+    https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.community.louvain.louvain_communities.html
+   https://networkit.github.io/dev-docs/notebooks/Community.html

**Pull request closed without merging**